package.json

```diff
@@ -6,8 +6,7 @@
   "main": "index.js",
   "scripts": {
     "pretest": "jshint *.js test",
-    "test": "istanbul test -- _mocha -R spec",
-    "posttest": "test -z $npm_config_coverage || istanbul report"
+    "test": "nyc -r lcov _mocha -R spec"
```

Comment:
istanbul is deprecated, as is npm_config_coverage, which doesn't get set anymore.  nyc is fast enough that it doesn't add enough overhead to the test loop to matter.